web/ui/react-app/src/Navbar.tsx

```diff
@@ -14,7 +14,11 @@ import {
 } from 'reactstrap';
 import PathPrefixProps from './types/PathPrefixProps';
 
-const Navigation: FC<PathPrefixProps> = ({ pathPrefix }) => {
+interface ConsolesLinkProps {
```

Comment:
Last nit: convention-wise, if this were a shared props type defined in one place only and reused from multiple components, then we would name it `ConsolesLinkProps`. Otherwise if it's a props type defined specifically for one component, we follow the naming convention `<Component>Props`, so `NavigationProps` here. There could be a case for making a shared props type out of it (like for `PathPrefixProps`), but IMO as long as it's only really used in one place (+ App pass-through), I prefer it defined twice, but with the component props naming convention.

Reply:
@juliusv Will `NavbarConsolesLink` and `AppConsolesLink` work fine?